Assets/MRTK/Examples/Demos/EyeTracking/Scenes/EyeTrackingDemo-04-TargetPositioning.unity

```diff
@@ -310,13 +324,13 @@ MonoBehaviour:
   m_margin: {x: 0, y: 0, z: -23.064568, w: -21.263912}
   m_textInfo:
     textComponent: {fileID: 91249384}
-    characterCount: 124
+    characterCount: 0
     spriteCount: 0
-    spaceCount: 27
-    wordCount: 25
+    spaceCount: 0
+    wordCount: 0
     linkCount: 0
-    lineCount: 5
-    pageCount: 1
+    lineCount: 0
+    pageCount: 0
```

Comment:
Intentionally included? Want to make sure there isn't any text being wiped out here